### PR TITLE
Adding Additional Testing

### PR DESCRIPTION
**Pull request by emarinier**
Re-enables various skipped components, tests their outputs, and adds a test profile test.

### CODE REVIEW

**Review by mattheww95 — APPROVED** [review]
Thank you for cleaning up the TODO's!

File: conf/test.config [hunk 1]
```diff
@@ -15,15 +15,88 @@ params {
...
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input = "https://raw.githubusercontent.com/phac-nml/mikrokondo/dev/tests/data/samplesheets/samplesheet-small-assembly.csv"
+    outdir = "results"
 
-    // Genome references
-    genome = 'R64-1-1'
+    platform = "illumina"
+
+    mash {
```
Comment (apetkau) [inline conf/test.config:28]: Thank so much for looking into this and including these parameters.

Have you tried only setting the parameters you need with dotted notation? As in:

```
mash.mash_sketch = "https://github.com/phac-nml/mikrokondo/raw/dev/tests/data/databases/campy-staph-ecoli.msh"
```

Could you maybe try this out, to see if we can avoid having duplicate copies of all parameters here (that may become out-of-sync)?

Alternatively, I wonder if the full test profile itself could remove the `params { }` block and set everything with dotted notation. As in:

```
params.mash.mash_sketch = "https://github.com/phac-nml/mikrokondo/raw/dev/tests/data/databases/campy-staph-ecoli.msh"
```

**Review by apetkau — COMMENTED** [review]
Thanks so much Eric. This looks awesome 😄 

One comment I have below that I'm wondering you could test out?

File: conf/test.config (hunk 1 above)
Comment (emarinier) [reply conf/test.config:28]: Oh sure. I can try it and see if it works.

Comment (emarinier) [reply conf/test.config]: Yup, you were right. It works. Updated in: 865fe4dfacdb6a998c581462e47205ff09b3cbd6

Comment (apetkau) [reply conf/test.config]: Awesome. Thanks so much 😄

**Review by apetkau — APPROVED** [review]
All looks good to me. Thanks so much Eric.

**Pull request merged**